build.sbt

```diff
@@ -78,3 +78,5 @@ assembly/assemblyMergeStrategy ~= { old =>
     case x => old(x)
   }
 }
+
+assembly/assemblyJarName := s"${name.value}-assembly.jar"
```

Comment:
Are you sure this change doesn't produce `*-assembly-assembly.jar`?

Reply:
Yes.  In the console output from the test build above:
```
[info] Built: /home/ubuntu/workspace/geo-export-branch/target/scala-2.12/geo-export-assembly.jar
```

Reply:
Isn't assembly added automatically? If it isn't, why add `-assembly` at all? The system now dynamically finds jar name.

Also, for context, does this mirror the approach I initially advocated in favor of (@rjmac's suggestion to replicate FFM's overriding of assemblyJarPath)?

Reply:
Yes, this approach mirrors FFM's.  I got the idea from there and confirmed that it has the effect of producing the build artifact named without the scala version, which is what is expected by the Dockerfile.